src/orders/views/OrderDraftList/OrderDraftList.tsx

```diff
@@ -79,7 +80,10 @@ export const OrderDraftList: React.FC<OrderDraftListProps> = ({ params }) => {
     onCompleted: handleCreateOrderCreateSuccess
   });
 
-  const { channel } = useAppChannel();
+  const { channel, availableChannels } = useAppChannel();
+  const [channelPickerDialogOpen, setChannelPickerDialogOpen] = React.useState(
+    false
+  );
```

Comment:
To maintain consistency with the rest of the project I recommend using query params to open this dialog.